src/components/player/player.js

```diff
@@ -1,11 +1,77 @@
-import React from "react"
+import React, { useEffect, useState } from "react"
 import { PlayButton } from "react-soundplayer/components"
 import { withCustomAudio } from "react-soundplayer/addons"
 
+import fetchJsonp from "fetch-jsonp"
+
 const Player = withCustomAudio(props => {
+  const [streamData, setStreamData] = useState({ streamstatus: 1 })
+  const [mixcloudData, setMixcloudData] = useState("")
+
+  const fetchMixcloudStream = url => {
+    if (!url) url = "https://api.mixcloud.com/entropyfm/feed/"
```

Comment:
We should put this url somewhere so that it's not hardcoded inline. Not sure if an environment var is necessary, but perhaps alongside siteMetadata?